pontoon/base/static/css/translate.css

```diff
@@ -1210,7 +1210,8 @@ body > header aside p {
 #editor p#original,
 #editor p.original,
 #ftl-original .main-value li,
-#editor p.translation {
+#editor p.translation,
+#editor p.translation-diff {
```

Comment:
I wasn't sure about text-align that why I've created an additional rule. However, this also looks fine for me (I'm a css barbarian) and works locally.